Auto update failing for proton-bridge

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [proton-bridge](https://github.com/termux/termux-packages/blob/3c9ccd0c6c75c5a7a76df7e3344a97605eec75c9/packages/proton-bridge) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating proton-bridge [Current version: 3.16.0]
INFO: package being updated to 3.18.0.
INFO: Trying to build package.
INFO: Package proton-bridge exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building proton-bridge for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  22747      0 --:--:-- --:--:-- --:--:-- 22723
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3265      0 --:--:-- --:--:-- --:--:--  3279
gpg: Signature made Sat 22 Mar 2025 01:02:04 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  57713      0 --:--:-- --:--:-- --:--:-- 57662
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3754      0 --:--:-- --:--:-- --:--:--  3769
gpg: Signature made Fri 21 Mar 2025 08:45:07 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  55609      0 --:--:-- --:--:-- --:--:-- 55829
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3752      0 --:--:-- --:--:-- --:--:--  3769
gpg: Signature made Fri 21 Mar 2025 08:48:04 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.7 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.18 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.18 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.45 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.45 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.24 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2025.02.25 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2025.02.25 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.4.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.4.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.7.0 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.4 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.4 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.49.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.49.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.49.1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.47 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.16.1 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.16.1 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.2 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.12 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0-1 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0-1 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.6.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.12 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.12 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.12.9 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.12.9 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.12.9 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency resolv-conf@1.3 if necessary...
Found resolv-conf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting resolv-conf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency glib@2.84.0-1 if necessary...
Found glib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting glib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency glib-cross@2.84.0-1 if necessary...
Found glib-cross in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting glib-cross to /home/builder/.termux-build/_cache-all...
Downloading dependency glib-bin@2.84.0-1 if necessary...
Found glib-bin in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting glib-bin to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-selinux@14.0.0.11 if necessary...
Found libandroid-selinux in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-selinux to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@27c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgmp@6.3.0-1 if necessary...
Found libgmp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgmp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency coreutils@9.6-1 if necessary...
Found coreutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting coreutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.65.0 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp3@1.8.0 if necessary...
Found libnghttp3 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp3 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.1 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.12.1 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.12.1 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dash@0.5.12 if necessary...
Found dash in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting dash to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dialog@1.3-20240307-0 if necessary...
Found dialog in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting dialog to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency diffutils@3.11 if necessary...
Found diffutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting diffutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency findutils@4.10.0 if necessary...
Found findutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting findutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmpfr@4.2.1 if necessary...
Found libmpfr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libmpfr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gawk@5.3.0 if necessary...
Found gawk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gawk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency grep@3.11 if necessary...
Found grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gzip@1.13 if necessary...
Found gzip in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gzip to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency less@668 if necessary...
Found less in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting less to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency procps@3.3.17-5 if necessary...
Found procps in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting procps to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency psmisc@23.7 if necessary...
Found psmisc in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting psmisc to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sed@4.9-1 if necessary...
Found sed in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sed to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-glob@0.6-2 if necessary...
Found libandroid-glob in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-glob to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tar@1.35 if necessary...
Found tar in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tar to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-am@0.8.0-1 if necessary...
Found termux-am in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-am to /home/builder/.termux-build/_cache-all...
Downloading dependency termux-am-socket@1.5.0 if necessary...
Found termux-am-socket in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-am-socket to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-core@0.1.0 if necessary...
Found termux-core in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-core to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-exec@1:2.0.0 if necessary...
Found termux-exec in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-exec to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libblkid@2.40.2-3 if necessary...
Found libblkid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libblkid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmount@2.40.2-3 if necessary...
Found libmount in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libmount to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsmartcols@2.40.2-3 if necessary...
Found libsmartcols in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsmartcols to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libuuid@2.40.2-3 if necessary...
Found libuuid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libuuid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libfdisk@2.40.2-3 if necessary...
Found libfdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libfdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcap-ng@2:0.8.5 if necessary...
Found libcap-ng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcap-ng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency util-linux@2.40.2-3 if necessary...
Found util-linux in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting util-linux to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency uuid-utils@2.40.2-3 if necessary...
Found uuid-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting uuid-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency mount-utils@2.40.2-3 if necessary...
Found mount-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting mount-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fdisk@2.40.2-3 if necessary...
Found fdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency blk-utils@2.40.2-3 if necessary...
Found blk-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting blk-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-tools@1.45.0 if necessary...
Found termux-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting termux-tools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bash@5.2.37-2 if necessary...
Found bash in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bash to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency binutils-bin@2.43.1 if necessary...
Found binutils-bin in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting binutils-bin to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency zstd@1.5.7 if necessary...
Found zstd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zstd to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pzstd@1.5.7 if necessary...
Found pzstd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pzstd to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency binutils-libs@2.43.1 if necessary...
Found binutils-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting binutils-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency binutils-gold@2.43.1 if necessary...
Found binutils-gold in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting binutils-gold to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency binutils-cross@2.43.1 if necessary...
Found binutils-cross in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting binutils-cross to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency binutils@2.43.1 if necessary...
Found binutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting binutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ldd@0.3 if necessary...
Found ldd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ldd to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gobject-introspection@1.84.0 if necessary...
Found gobject-introspection in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gobject-introspection to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency g-ir-scanner@1.84.0 if necessary...
Found g-ir-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting g-ir-scanner to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgpg-error@1.50 if necessary...
Found libgpg-error in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgpg-error to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgcrypt@1.11.0 if necessary...
Found libgcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libjpeg-turbo@3.1.0 if necessary...
Found libjpeg-turbo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libjpeg-turbo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libjpeg-turbo-progs@3.1.0 if necessary...
Found libjpeg-turbo-progs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libjpeg-turbo-progs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libtiff@4.7.0 if necessary...
Found libtiff in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libtiff to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libtiff-utils@4.7.0 if necessary...
Found libtiff-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libtiff-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdk-pixbuf@2.42.12 if necessary...
Found gdk-pixbuf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdk-pixbuf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-execinfo@0.1-1 if necessary...
Found libandroid-execinfo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-execinfo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-shmem@0.5 if necessary...
Found libandroid-shmem in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-shmem to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzo@2.10-3 if necessary...
Found liblzo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpixman@0.44.2 if necessary...
Found libpixman in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpixman to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcairo@1.18.4 if necessary...
Found libcairo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcairo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgraphite@1.3.14-2 if necessary...
Found libgraphite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgraphite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libicu@77.1 if necessary...
Found libicu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libicu to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency icu-devtools@77.1 if necessary...
Found icu-devtools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting icu-devtools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency harfbuzz@10.4.0-1 if necessary...
Found harfbuzz in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting harfbuzz to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency harfbuzz-utils@10.4.0-1 if necessary...
Found harfbuzz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting harfbuzz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency harfbuzz-icu@10.4.0-1 if necessary...
Found harfbuzz-icu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting harfbuzz-icu to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libaom@3.12.0 if necessary...
Found libaom in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libaom to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency aom-tools@3.12.0 if necessary...
Found aom-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting aom-tools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libdav1d@1.5.1 if necessary...
Found libdav1d in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libdav1d to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libde265@1.0.15 if necessary...
Found libde265 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libde265 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency librav1e@0.7.1-3 if necessary...
Found librav1e in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting librav1e to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency rav1e@0.7.1-3 if necessary...
Found rav1e in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting rav1e to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libx265@4.1 if necessary...
Found libx265 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx265 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency x265@4.1 if necessary...
Found x265 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting x265 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libheif@1.19.7 if necessary...
Found libheif in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libheif to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libheif-progs@1.19.7 if necessary...
Found libheif-progs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libheif-progs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency giflib@5.2.2 if necessary...
Found giflib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting giflib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency giflib-utils@5.2.2 if necessary...
Found giflib-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting giflib-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwebp@1.5.0 if necessary...
Found libwebp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libwebp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgd@1:2.3.3-4 if necessary...
Found libgd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgd to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgts@0.7.6-1 if necessary...
Found libgts in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgts to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libtool@2.5.4 if necessary...
Found libtool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libtool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libltdl@2.5.4 if necessary...
Found libltdl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libltdl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2@2.13.6-1 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.13.6-1 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.13.6-1 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.13.6-1 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fribidi@1.0.16 if necessary...
Found fribidi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fribidi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pango@1.56.3 if necessary...
Found pango in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pango to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency librsvg@2.60.0 if necessary...
Found librsvg in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting librsvg to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency graphviz@12.2.1 if necessary...
Found graphviz in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting graphviz to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency valac@0.56.18 if necessary...
Found valac in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/valac/valac_0.56.18_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 18 1629k   18  303k    0     0   260k      0  0:00:06  0:00:01  0:00:05  259k
100 1629k  100 1629k    0     0  1095k      0  0:00:01  0:00:01 --:--:-- 1095k
extracting valac to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency valadoc@0.56.18 if necessary...
Found valadoc in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/valadoc/valadoc_0.56.18_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  338k  100  338k    0     0  4856k      0 --:--:-- --:--:-- --:--:-- 4899k
extracting valadoc to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency valadoc-static@0.56.18 if necessary...
Found valadoc-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/valadoc-static/valadoc-static_0.56.18_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  365k  100  365k    0     0  6089k      0 --:--:-- --:--:-- --:--:-- 6189k
extracting valadoc-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsecret@0.21.7 if necessary...
Found libsecret in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libsecret/libsecret_0.21.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 27  140k   27 39265    0     0  40489      0  0:00:03 --:--:--  0:00:03 40479
100  140k  100  140k    0     0   122k      0  0:00:01  0:00:01 --:--:--  122k
extracting libsecret to /home/builder/.termux-build/_cache-aarch64...
Downloading proton-bridge@3.18.0 git source from 'git+https://github.com/ProtonMail/proton-bridge' if necessary...
Downloading git source with branch 'v3.18.0' from 'https://github.com/ProtonMail/proton-bridge'
Cloning into '/home/builder/.termux-build/proton-bridge/cache/tmp-checkout'...
Note: switching to '0800aeea50ee8a474e64355a4b7df124c0bcee91'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

~/.termux-build/proton-bridge/cache/tmp-checkout ~/.termux-build/proton-bridge/cache
Submodule 'submodules/vcpkg' (https://github.com/Microsoft/vcpkg.git) registered for path 'extern/vcpkg'
Cloning into '/home/builder/.termux-build/proton-bridge/cache/tmp-checkout/extern/vcpkg'...
From https://github.com/Microsoft/vcpkg
 * branch              fba75d09065fcc76a25dcf386b1d00d33f5175af -> FETCH_HEAD
Submodule path 'extern/vcpkg': checked out 'fba75d09065fcc76a25dcf386b1d00d33f5175af'
~/.termux-build/proton-bridge/cache
Applying patch: disable-updater.patch
cd ./utils/ && ./credits.sh bridge
go build -tags='' -ldflags '-X github.com/ProtonMail/proton-bridge/v3/internal/constants.Version=3.18.0+git -X github.com/ProtonMail/proton-bridge/v3/internal/constants.Revision=0800aeea50 -X github.com/ProtonMail/proton-bridge/v3/internal/constants.Tag=v3.18.0 -X github.com/ProtonMail/proton-bridge/v3/internal/constants.BuildTime=2025-03-22T01:04:34+0000 -X "github.com/ProtonMail/proton-bridge/v3/internal/constants.FullAppName=Proton Mail Bridge" -X github.com/ProtonMail/proton-bridge/v3/internal/constants.BuildEnv=dev' -o "proton-bridge" "./cmd/Desktop-Bridge/"
go: downloading github.com/bradenaw/juniper v0.12.0
go: downloading github.com/sirupsen/logrus v1.9.2
go: downloading github.com/Masterminds/semver/v3 v3.2.0
go: downloading github.com/ProtonMail/gluon v0.17.1-0.20250116113909-2ebd96ec0bc2
go: downloading github.com/ProtonMail/go-autostart v0.0.0-20210130080809-00ed301c8e9a
go: downloading github.com/ProtonMail/gopenpgp/v2 v2.8.2-proton
go: downloading github.com/elastic/go-sysinfo v1.11.2-0.20231129083954-35e55cd2a542
go: downloading github.com/getsentry/sentry-go v0.15.0
go: downloading github.com/jeandeaual/go-locale v0.0.0-20220711133428-7de61946b173
go: downloading github.com/allan-simon/go-singleinstance v0.0.0-20210120080615-d0997106ab37
go: downloading github.com/hashicorp/go-multierror v1.1.1
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/pkg/profile v1.7.0
go: downloading github.com/urfave/cli/v2 v2.24.4
go: downloading golang.org/x/exp v0.0.0-20230522175609-2e198f4a06a1
go: downloading golang.org/x/sys v0.29.0
go: downloading github.com/ProtonMail/go-crypto v1.1.4-proton
go: downloading github.com/ProtonMail/go-mime v0.0.0-20230322103455-7d82a3887f2f
go: downloading github.com/ProtonMail/go-proton-api v0.4.1-0.20250217140732-2e531f21de4c
go: downloading github.com/emersion/go-imap v1.2.1
go: downloading github.com/LBeernaertProton/resty/v2 v2.0.0-20231129100320-dddf8030d93a
go: downloading github.com/golang/mock v1.6.0
go: downloading github.com/0xAX/notificator v0.0.0-20220220101646-ee9b8921e557
go: downloading github.com/google/uuid v1.3.0
go: downloading github.com/google/go-cmp v0.6.0
go: downloading github.com/miekg/dns v1.1.50
go: downloading google.golang.org/grpc v1.56.3
go: downloading google.golang.org/protobuf v1.33.0
go: downloading github.com/abiosoft/ishell v2.0.0+incompatible
go: downloading github.com/fatih/color v1.13.0
go: downloading github.com/vmihailenco/msgpack/v5 v5.3.5
go: downloading github.com/docker/docker-credential-helpers v0.8.1
go: downloading github.com/godbus/dbus v4.1.0+incompatible
go: downloading github.com/cuthix/go-keychain v0.0.0-20240103134243-0b6a41580b77
go: downloading github.com/hashicorp/errwrap v1.1.0
go: downloading github.com/felixge/fgprof v0.9.3
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.2
go: downloading github.com/xrash/smetrics v0.0.0-20201216005158-039620a65673
go: downloading golang.org/x/crypto v0.32.0
go: downloading github.com/cloudflare/circl v1.5.0
go: downloading github.com/emersion/go-sasl v0.0.0-20220912192320-0145f2c60ead
go: downloading github.com/ProtonMail/go-smtp v0.0.0-20231109081432-2b3d50599865
go: downloading github.com/pbnjay/memory v0.0.0-20210728143218-7b4eea64cf58
go: downloading github.com/ProtonMail/go-message v0.13.1-0.20240919135104-3bc88e6a9423
go: downloading golang.org/x/text v0.21.0
go: downloading golang.org/x/net v0.34.0
go: downloading github.com/ProtonMail/go-srp v0.0.7
go: downloading github.com/PuerkitoBio/goquery v1.8.1
go: downloading gitlab.com/c0b/go-ordered-json v0.0.0-20201030195603-febf46534d5a
go: downloading github.com/emersion/go-vcard v0.0.0-20230331202150-f3d26859ccd3
go: downloading github.com/golang/protobuf v1.5.3
go: downloading google.golang.org/genproto v0.0.0-20230410155749-daa745c078e1
go: downloading golang.org/x/sync v0.10.0
go: downloading github.com/mattn/go-colorable v0.1.13
go: downloading github.com/mattn/go-isatty v0.0.19
go: downloading github.com/vmihailenco/tagparser/v2 v2.0.0
go: downloading github.com/abiosoft/readline v0.0.0-20180607040430-155bce2042db
go: downloading github.com/flynn-archive/go-shlex v0.0.0-20150515145356-3f9db97f8568
go: downloading howett.net/plist v1.0.0
go: downloading github.com/joeshaw/multierror v0.0.0-20140124173710-69b34d4ec901
go: downloading github.com/prometheus/procfs v0.12.0
go: downloading github.com/russross/blackfriday/v2 v2.1.0
go: downloading github.com/google/pprof v0.0.0-20211214055906-6f57359322fd
go: downloading github.com/mattn/go-sqlite3 v1.14.22
go: downloading github.com/pierrec/lz4/v4 v4.1.17
go: downloading github.com/jaytaylor/html2text v0.0.0-20211105163654-bc68cce691ba
go: downloading github.com/emersion/go-textwrapper v0.0.0-20200911093747-65d896831594
go: downloading github.com/andybalholm/cascadia v1.3.2
go: downloading github.com/olekukonko/tablewriter v0.0.5
go: downloading github.com/ssor/bom v0.0.0-20170718123548-6386211fdfcf
go: downloading github.com/mattn/go-runewidth v0.0.14
go: downloading github.com/ProtonMail/bcrypt v0.0.0-20211005172633-e235017c1baf
go: downloading github.com/cronokirby/saferith v0.33.0
go: downloading github.com/rivo/uniseg v0.4.2
# github.com/ProtonMail/proton-bridge/v3/internal/bridge
internal/bridge/bridge.go:460:30: undefined: updater.ErrVersionFileDownloadOrVerify
internal/bridge/updates.go:323:31: undefined: updater.ErrReleaseUpdatePackageMissing
make: *** [Makefile:111: proton-bridge] Error 1
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-03-22 01:41:22 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/14003094870">14003094870</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/proton-bridge/build.sh
+++ b/packages/proton-bridge/build.sh
@@ -1,11 +1,11 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/ProtonMail/proton-bridge
 TERMUX_PKG_DESCRIPTION="ProtonMail Bridge application"
 TERMUX_PKG_LICENSE="GPL-3.0"
-TERMUX_PKG_VERSION="3.16.0"
+TERMUX_PKG_VERSION="3.18.0"
 TERMUX_PKG_SRCURL=git+https://github.com/ProtonMail/proton-bridge
 TERMUX_PKG_GIT_BRANCH=v${TERMUX_PKG_VERSION}
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_DEPENDS=libsecret
+TERMUX_PKG_DEPENDS="glib, libsecret"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_TAG_TYPE=latest-release-tag
 TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+\.\d+"

--- a/packages/proton-bridge/disable-updater.patch
+++ b/packages/proton-bridge/disable-updater.patch
@@ -12,7 +12,7 @@
  import (
 --- /dev/null
 +++ b/internal/updater/updater_android.go
-@@ -0,0 +1,76 @@
+@@ -0,0 +1,86 @@
 +// Copyright (c) 2023 Proton AG
 +//
 +// This file is part of Proton Mail Bridge.
@@ -46,9 +46,11 @@
 +)
 +
 +var (
-+	ErrDownloadVerify         = errors.New("failed to download or verify the update")
-+	ErrInstall                = errors.New("failed to install the update")
-+	ErrUpdateAlreadyInstalled = errors.New("update is already installed")
++	ErrDownloadVerify              = errors.New("failed to download or verify the update")
++	ErrInstall                     = errors.New("failed to install the update")
++	ErrUpdateAlreadyInstalled      = errors.New("update is already installed")
++	ErrVersionFileDownloadOrVerify = errors.New("failed to download or verify the version file")
++	ErrReleaseUpdatePackageMissing = errors.New("release update package is missing")
 +)
 +
 +type Downloader interface {
@@ -78,11 +80,19 @@
 +	}
 +}
 +
-+func (u *Updater) GetVersionInfo(ctx context.Context, downloader Downloader, channel Channel) (VersionInfo, error) {
++func (u *Updater) GetVersionInfoLegacy(ctx context.Context, downloader Downloader, channel Channel) (VersionInfoLegacy, error) {
++	return VersionInfoLegacy{}, errors.New("no updates available for this channel")
++}
++
++func (u *Updater) GetVersionInfo(ctx context.Context, downloader Downloader) (VersionInfo, error) {
 +	return VersionInfo{}, errors.New("no updates available for this channel")
 +}
 +
-+func (u *Updater) InstallUpdate(ctx context.Context, downloader Downloader, update VersionInfo) error {
++func (u *Updater) InstallUpdateLegacy(ctx context.Context, downloader Downloader, update VersionInfoLegacy) error {
++	return nil
++}
++
++func (u *Updater) InstallUpdate(ctx context.Context, downloader Downloader, release Release) error {
 +	return nil
 +}
 +

